api/templates/api.mustache

```diff
@@ -388,11 +388,11 @@ func (a *{{{classname}}}Service) {{nickname}}Execute(r {{#structPrefix}}{{&class
 			var v {{{dataType}}}
 			err = a.client.decode(&v, localVarBody, localVarHTTPResponse.Header.Get("Content-Type"))
 			if err != nil {
-				newErr.error = _fmt.Sprintf("%s%v", errorPrefix, err.Error())
+				newErr.error = _fmt.Sprintf("%s: %v", errorPrefix, err.Error())
 				return {{#returnType}}localVarReturnValue, {{/returnType}}newErr
 			}
+			v.SetMessage(errorPrefix)
```

Comment:
This is the change, right? And everything else is adapting to it?

Reply:
I think it would be valuable to still include the next of the error model. i.e. to show:
```
InfluxDB OSS-only command failed: path not found
```
instead of just:
```
InfluxDB OSS-only command failed
```
Even if it doesn't help users, it'll help us triage bug reports

Reply:
Yep this is pretty much the fundamental change! I updated the code to preserve & display the actual error contained in `v` along with the prefix; good idea.